### ISSUE TITLE
libc requirements in DEB

### ISSUE THREAD

**Issue by Feelemoon** [devops-kung-fu/bomber#145]
After upgrading to bomber 0.4.2 using the provided DEB, it does not start anymore. The following error is shown:

```
$ bomber scan --debug cdx.json
bomber: /lib64/libc.so.6: version `GLIBC_2.32' not found (required by bomber)
bomber: /lib64/libc.so.6: version `GLIBC_2.34' not found (required by bomber)
```

**Comment by djschleen** [comment]
Will check this out. What distro are you using?

**Comment by Feelemoon** [comment]
I'm using ubuntu:latest

**Comment by djschleen** [comment]
Hey @Feelemoon - I just installed the deb on a fresh ubuntu desktop and ran it without errors. With that said, it's not a container, but that rules out an issue with the deb file. I found there are some changes to go 1.20 that cause some compile issues (https://github.com/golang/go/issues/58550). I'll take a look at the build options for goreleaser and see if I can reproduce it with an ubuntu:latest docker image.
![image](https://user-images.githubusercontent.com/32048395/225931095-508266c9-6ddb-4a1b-b14e-b377214b82b4.png)
